sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -243,6 +243,14 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val PARQUET_READ_SCHEMA_LOCAL_THRESHOLD =
+    SQLConfigBuilder("spark.sql.parquet.readSchemaLocalThreshold")
+      .doc("Configures the maximum number of files to allow reading schema in driver-side. " +
+        "If the number of files exceeds this value, then schemas will be read then in parallel " +
+        "via another Spark distributed job.")
+      .intConf
+      .createWithDefault(32)
```

Comment:
I increased this to 32 identical with `spark.sql.sources.parallelPartitionDiscovery.threshold` because it seems this option is pretty similar with this.